.github/workflows/map-diff.yml

```diff
@@ -0,0 +1,88 @@
+name: Map Diff
+
+on:
+  pull_request:
+    paths:
+    - '**.map'
+
+jobs:
+  comment-map-diff:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3.0.0
+    - uses: actions/setup-node@v3.0.0
+      with:
+        node-version: '16'
+    - name: Package install
+      run: |
+        cd ./utils/wesnoth-map-diff
+        npm install
+    - name: Package build
+      run: |
+        cd ./utils/wesnoth-map-diff
+        npm run build:prod
+    - name: Get maps
+      id: get-maps
+      run: |
+        cd ./utils/wesnoth-map-diff
+
+        git fetch --depth=1 origin ${{ github.event.pull_request.base.sha }}
+
+        MAP_PATHS=$(git diff --name-status HEAD ${{ github.event.pull_request.base.sha }} | grep ".map$" | perl -pe 's/\w\s+//')
```

Comment:
You can just as well do `git diff --name-only HEAD ${{ github.event.pull_request.base.sha }} '*.map'`

Reply:
Thanks. I didn't know the flag `--name-only`. But I still need to use `grep`:
```
➜  wesnoth-map-diff git:(feat/add-wesnoth-map-diff) ✗ git diff --name-only HEAD 514b8bca9cf '*.map'
➜  wesnoth-map-diff git:(feat/add-wesnoth-map-diff) ✗ git diff --name-only HEAD 514b8bca9cf | grep ".map$"  
data/test.map
data/test2.map
```